### ISSUE TITLE
Building on iOS with the new architecture enabled fails. 

### ISSUE THREAD

**Issue by psquizzle** [Expensify/react-native-live-markdown#326]
I get the following errors but works with new arch not enabled. Using     "@expensify/react-native-live-markdown": "^0.1.63"

```
[RUN_FASTLANE] › Compiling @expensify/react-native-live-markdown Pods/RNLiveMarkdown » RNLiveMarkdown-dummy.m
[RUN_FASTLANE] › Compiling @expensify/react-native-live-markdown Pods/RNLiveMarkdown » RCTBaseTextInputView+Markdown.m
[RUN_FASTLANE]
❌  (ios/Pods/Headers/Public/React-Fabric/react/renderer/components/view/AccessibilityPrimitives.h:10:10)

   8 | #pragma once
   9 |
> 10 | #include <cinttypes>
     |          ^ 'cinttypes' file not found
  11 | #include <optional>
  12 | #include <string>
  13 | #include <vector>

[RUN_FASTLANE]
❌  (ios/Pods/Headers/Public/glog/glog/logging.h:42:10)

  40 | #include <string.h>
  41 | #include <time.h>
> 42 | #include <iosfwd>
     |          ^ 'iosfwd' file not found
  43 | #include <ostream>
  44 | #include <sstream>
  45 | #include <string>

[RUN_FASTLANE] › Compiling @expensify/react-native-live-markdown Pods/RNLiveMarkdown » RCTUITextView+Markdown.mm
[RUN_FASTLANE] › Compiling @expensify/react-native-live-markdown Pods/RNLiveMarkdown » RCTTextInputComponentView+Markdown.mm
[RUN_FASTLANE] › Compiling @expensify/react-native-live-markdown Pods/RNLiveMarkdown » RCTMarkdownUtils.mm
[RUN_FASTLANE] ▸ ** ARCHIVE FAILED **
[RUN_FASTLANE] ▸ The following build commands failed:
[RUN_FASTLANE] ▸ 	CompileC /Users/peter/Library/Developer/Xcode/DerivedData/snipmap-dichtoroovopmbcnvhbwvywypevy/Build/Intermediates.noindex/ArchiveIntermediates/snipmap/IntermediateBuildFilesPath/Pods.build/Debug-iphoneos/RNLiveMarkdown.build/Objects-normal/arm64/RCTBaseTextInputView+Markdown.o /var/folders/rl/9tgdr27n0nj7lm5mt4t58tr80000gn/T/eas-build-local-nodejs/dcc53e11-373f-49b3-9ddb-74cad5236a75/build/node_modules/@expensify/react-native-live-markdown/ios/RCTBaseTextInputView+Markdown.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'RNLiveMarkdown' from project 'Pods')
[RUN_FASTLANE] ▸ (1 failure)
[RUN_FASTLANE] ** ARCHIVE FAILED **
[RUN_FASTLANE]
[RUN_FASTLANE]
[RUN_FASTLANE] The following build commands failed:
[RUN_FASTLANE] 	CompileC /Users/peter/Library/Developer/Xcode/DerivedData/snipmap-dichtoroovopmbcnvhbwvywypevy/Build/Intermediates.noindex/ArchiveIntermediates/snipmap/IntermediateBuildFilesPath/Pods.build/Debug-iphoneos/RNLiveMarkdown.build/Objects-normal/arm64/RCTBaseTextInputView+Markdown.o /var/folders/rl/9tgdr27n0nj7lm5mt4t58tr80000gn/T/eas-build-local-nodejs/dcc53e11-373f-49b3-9ddb-74cad5236a75/build/node_modules/@expensify/react-native-live-markdown/ios/RCTBaseTextInputView+Markdown.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'RNLiveMarkdown' from project 'Pods')
[RUN_FASTLANE] (1 failure)
[RUN_FASTLANE] Exit status: 65
```

**Comment by tomekzaw** [comment]
@psquizzle Can you check if 0.1.66 works for you? We've just merged https://github.com/Expensify/react-native-live-markdown/pull/315 that should resolve this problem.

**Comment by psquizzle** [comment]
> @psquizzle Can you check if 0.1.66 works for you? We've just merged #315 that should resolve this problem.

So same error occurs when building with   "@expensify/react-native-live-markdown": "^0.1.66" installed.
> 10 | #include <cinttypes>
     |          ^ 'cinttypes' file not found
> 42 | #include <iosfwd>
     |          ^ 'iosfwd' file not found
> 10 | #include <cinttypes>
     |          ^ 'cinttypes' file not found

**Comment by tomekzaw** [comment]
@psquizzle Just to confirm, did you run `RCT_NEW_ARCH_ENABLED=1 pod install` after bumping version to 0.1.66?

**Comment by psquizzle** [comment]
> @psquizzle Just to confirm, did you run `RCT_NEW_ARCH_ENABLED=1 pod install` after bumping version to 0.1.66?

I am using the expo toolchain, and running npx expo prebuild --clean reinstalls pods and and generates the iOS folder dynamically. With the same error occurring. Turning off newArch the build works.

**Comment by tomekzaw** [comment]
@psquizzle We found the root cause of the problem, it was fixed in #331 and released in 0.1.67. Can you please check if it works for you?

**Comment by psquizzle** [comment]
Cheers, builds and runs now :)
